lib/neovim-plugin.nix

```diff
@@ -128,10 +130,8 @@ with lib; rec {
               require('${luaName}').setup(${toLuaObject cfg.settings})
             '';
           }
-          (optionalAttrs colorscheme {
-            # We use `mkDefault` here to let individual plugins override this option.
-            # For instance, setting it to `null` a specific way of setting the coloscheme.
-            colorscheme = lib.mkDefault name;
+          (optionalAttrs (isColorscheme && (colorscheme != null)) {
+            inherit colorscheme;
```

Comment:
Otherwise, we can do:
```nix
(optionalAttrs isColorscheme {
  colorscheme = mkDefault colorscheme;
})
```
Then, the plugins that want to manually unset it would add 
```nix
colorscheme = null;
```
to their `extraConfig`.

@traxys which option do you prefer ?

Reply:
I don't really care about either way

Reply:
Ok fine !